docs/installing.md

```diff
@@ -84,26 +64,25 @@ You can now:
 ansible-playbook -i inventory/hosts setup.yml --tags=start
 ```
 
-Proceed to [Maintaining your setup in the future](#2-maintaining-your-setup-in-the-future) and [Finalize the installation](#3-finalize-the-installation)
-
+## Finalize the installation
 
-## 2. Maintaining your setup in the future
-
-Feel free to **re-run the setup command any time** you think something is off with the server configuration. Ansible will take your configuration and update your server to match.
-
-Note that if you remove components from `vars.yml`, or if we switch some component from being installed by default to not being installed by default anymore, you'd need to run the setup command with `--tags=setup-all` instead of `--tags=install-all`. See [Playbook tags introduction](#playbook-tags-introduction)
+Now that services are running, you need to **finalize the installation process** (required for federation to work!) by [Configuring Service Discovery via .well-known](configuring-well-known.md).
 
-A way to invoke these `ansible-playbook` commands with less typing in the future is to use [just](https://github.com/casey/just) to run them: `just install-all` or `just setup-all`. See [our `justfile`](../justfile) for more information.
+If you need the base domain for anything else (such as hosting a website), you have to configure it manually, following the procedure described on the linked documentation.
 
+However, if you do not need the base domain (`example.com`) for anything else, the easiest way of configuring it is to [serve the base domain](configuring-playbook-base-domain-serving.md) from the integrated web server. It will enable you to use a Matrix user identifier like `@<username>:example.com` while hosting services on a subdomain like `matrix.example.com`.
```

Comment:
I think this should be correct according to [this line](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7a77d84276ca3f608487757738e1a4821a3eb078/docs/configuring-well-known.md?plain=1#L16), but I am not perfectly sure.